apollo-adminservice/src/main/scripts/startup.sh

```diff
@@ -53,6 +56,35 @@ if [ "$windows" == "1" ] && [[ -n "$JAVA_HOME" ]] && [[ -x "$JAVA_HOME/bin/java"
     echo "Windows new JAVA_HOME is: $JAVA_HOME"
 fi
 
+if [[ -n "$JAVA_HOME" ]] && [[ -x "$JAVA_HOME/bin/java" ]];  then
+    _java="$JAVA_HOME/bin/java"
+else
+    echo "no executable java command found"
+    exit 1
+fi
+
+if [[ "$_java" ]]; then
+    version=$("$_java" -version 2>&1 | awk -F '"' '/version/ {print $2}')
+    echo version "$version"
```

Comment:
Is it necessary to print out `version 1.8.0_60` and `java version >= 8` as they look more like debug messages?

Reply:
Good catch! I'll remove them and only leave necessary messages